cse-adapter-app/src/test/java/com/farao_community/farao/cse/adapter/app/CseAdapterListenerTest.java

```diff
@@ -41,12 +54,74 @@ void testIdccSuccess() {
         processFiles.add(new ProcessFileDto("NTC2-CH", ProcessFileStatus.VALIDATED, "ch-ntc2", timestamp, "file://ch-ntc2.xml"));
         processFiles.add(new ProcessFileDto("NTC2-FR", ProcessFileStatus.VALIDATED, "fr-ntc2", timestamp, "file://fr-ntc2.xml"));
         processFiles.add(new ProcessFileDto("NTC2-SI", ProcessFileStatus.VALIDATED, "si-ntc2", timestamp, "file://si-ntc2.xml"));
-        TaskDto taskDto = new TaskDto(id, timestamp, TaskStatus.READY, processFiles);
+        return new TaskDto(id, timestamp, TaskStatus.READY, processFiles);
+    }
+
+    private TaskDto getD2ccTaskDto() {
+        UUID id = UUID.randomUUID();
+        LocalDateTime timestamp = LocalDateTime.parse("2021-12-07T14:30");
```

Comment:
Don't forget that we are trying to move every thing in back-end to OffsetDateTime

Reply:
If you prefer we handle that in another PR, no harm in that!